tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2510,6 +2510,29 @@ public function userSwitchesTheSystemLanguageUsingGraphApi(string $user, string
 		);
 	}
 
+	/**
+	 *
+	 * @param string $user
+	 * @param bool $waitForUserDeletion
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public function getSharesSharedWithMe(string $user, bool $waitForUserDeletion = false): ResponseInterface {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		if ($waitForUserDeletion) {
+			// it takes few second to delete a user so wating until user is deleted
+			sleep(1);
```

Comment:
add some comments mentioning the env vars like in
https://github.com/owncloud/ocis/blob/f60e392d5977e45c760aea760a4756fd7671cfe2/tests/acceptance/features/bootstrap/GraphContext.php#L2542-L2547